.github/workflows/main.yml

```
@@ -46,3 +63,36 @@ jobs:
           cd ${NAME}
           zip ${NAME}.zip lambda_handler.py
           aws s3 cp ${NAME}.zip s3://${S3_BUCKET}/${S3_KEY}/${NAME}.zip
+  upload-elastio-s3-changelog:
+    runs-on:
+      - ubuntu-latest
```

Make it simpler, refactor the other job as well
```suggestion
    runs-on: ubuntu-latest
```